scipy/spatial/transform/rotation.py

```diff
@@ -657,3 +658,32 @@ def __mul__(self, other):
         if self._single and other._single:
             result = result[0]
         return self.__class__(result, normalized=True)
+
+    def apply(self, points):
```

Comment:
What about `inverse=False` flag?

Implementing would be really simple --- adding `.T` attribute to the DCM.